apiary.apib

```diff
@@ -2347,6 +2347,68 @@ If there is no column datatype specified (column.definition.type or column.baset
             }
         }
 
+## Create Table with Configuration [/v2/storage/buckets/{bucket_id}/tables-with-configuration]
```

Comment:
Ještě v detailu tabulky (GET tables/tableId) musí přibýt ID configurace

Reply:
Proc by melo, je to obycejna typovana tabulka, jen mela jinej zpusob vytvoreni, ne?
Navic potom by se `configurationId` vracelo ve vsech requestech pro TableDetail...

Reply:
https://keboola.atlassian.net/browse/KBC-3000